### PR TITLE
#191: improve logging in case of Exception

### PR DESCRIPTION
**Pull request by andrii-kovalenko-ct**
This is just minor intermediate improvement before implementing normal workflow in issue #164 

So far we just implemented more verbose logging for Payone connection issues and CTP connection issues.

### CODE REVIEW

File: service/src/main/java/com/commercetools/pspadapter/payone/PaymentHandler.java [hunk 1]
```diff
@@ -63,11 +63,12 @@ public PaymentHandleResult handlePayment(@Nonnull final String paymentId) {
                     paymentDispatcher.dispatchPayment(paymentWithCartLike);
                     return new PaymentHandleResult(HttpStatusCode.OK_200);
                 } catch (final ConcurrentModificationException cme) {
+                    logger.warn("ConcurrentModificationException on payment [{}]. Retry #{}", paymentId, i + 1);
```
Comment (butenkor) [inline service/src/main/java/com/commercetools/pspadapter/payone/PaymentHandler.java]: `Exception on dispatchPayment with id '123'. Retry 3 of 20`

Comment (andrii-kovalenko-ct) [reply service/src/main/java/com/commercetools/pspadapter/payone/PaymentHandler.java]: e4a8c607

**Comment by andrii-kovalenko-ct** [comment]
@butenkor 
static loggers added. Could the PR be approved?

**Comment by andrii-kovalenko-ct** [comment]
close #164

**Pull request merged**